### PR TITLE
Insulated double corner

### LINKED ISSUE #408: Double corner insulated wire suggestion
Hi,

It seems to me that a double corner wire would make the mesecon insulated wires more complete. 

other_wire

This would be specially useful when wanting to shift to the side a couple of parallel wires in a compact way. For example.

screenshot_20180530_111256

What do you think of this?

### PR DESCRIPTION
**Pull request by numberZero**
Fixes #408. Also makes the regular corner curved.
screenshot_20200816_123338

### CODE REVIEW

**Comment by SmallJoker** [comment]
> Should the regular corner use the same shape?

My preference would be to use this corner style for all corners, even if it increases rendering complexity.
I like the research you did about various shapes, well done!

**Comment by numberZero** [comment]
> use this corner style for all corners

Done.

File: mesecons_extrawires/corner.lua [hunk 1]
```diff
@@ -25,14 +18,11 @@ local corner_get_rules = function (node)
 end
 
 minetest.register_node("mesecons_extrawires:corner_on", {
-	drawtype = "nodebox",
+	drawtype = "mesh",
+	mesh = "mesecons_extrawires_corner.obj",
```
Comment (SmallJoker) [inline mesecons_extrawires/corner.lua:22]: `2020-08-16 13:07:15: ERROR[Main]: Client::getMesh(): Mesh not found: "mesecons_extrawires_corner.obj"`

File: mesecons_extrawires/corner.lua [hunk 2]
```diff
@@ -55,15 +45,12 @@ minetest.register_node("mesecons_extrawires:corner_on", {
 })
 
 minetest.register_node("mesecons_extrawires:corner_off", {
-	drawtype = "nodebox",
+	drawtype = "mesh",
 	description = "Insulated Mesecon Corner",
+	mesh = "mesecons_extrawires_corner.obj",
```
Comment (SmallJoker) [inline mesecons_extrawires/corner.lua:50]: `2020-08-16 13:07:15: ERROR[Main]: Client::getMesh(): Mesh not found: "mesecons_extrawires_corner.obj"`

File: mesecons_extrawires/corner.lua (hunk 1 above)
Comment (numberZero) [reply mesecons_extrawires/corner.lua:22]: oops...

**Comment by SmallJoker** [comment]
grafik

Looks good now.

**Comment by VanessaE** [comment]
@SmallJoker perfect! :grin:

**Comment by Jeija** [comment]
@numberZero, the new double corner is not documented yet on [mesecons.net/items.html](http://mesecons.net/items.html) and the single corner preview image is still the old one. If you could take care of documenting the double corner and updating the preview image of the single corner, I can update the website.

The item documentation is in this repository in `mesecons_extrawires/doc`. To make the new item appear on mesecons.net, you'll also have to update `documentation.json` in the repository root directory.

**Comment by numberZero** [comment]
Okay, will do that before the release.

**Pull request merged**